### ISSUE TITLE
Potential bug when loading USG model

### ISSUE THREAD

**Issue by DataGeoranger** [modflowpy/flopy#1072]
Hey Team,

I tried to use the flopy.modflow.Modflow.load to load my existing robust USG model. It gave me error says " not enough values to unpack (expected 3, got 1)" and screenshot below. I further confirmed the error using USGS test example - folder '01B_nestedgrid_gnc'. Could you please help? Thanks!

Jack Wang

![image](https://user-images.githubusercontent.com/47303462/110269292-64058180-7f91-11eb-9bdc-354b81f57eb6.png)

**Comment by DataGeoranger** [comment]
Just want to clarify, the USGS example I referred above is an USG model when I downloaded the USG program from USGS website. The version is mfusg1_5 and under the folder called "test". "..\mfusg1_5\test".

**Comment by cnicol-gwlogic** [comment]
> Just want to clarify, the USGS example I referred above is an USG model when I downloaded the USG program from USGS website. The version is mfusg1_5 and under the folder called "test". "..\mfusg1_5\test".  
> 3.8.5 (default, Sep  3 2020, 21:29:08) [MSC v.1916 64 bit (AMD64)]
> numpy version: 1.19.2
> matplotlib version: 3.3.1
> pandas version: 1.1.3
> flopy version: 3.3.4
> @langevin-usgs Hey Chris, just wonder if you could help with this issue. Thank you!

I've been meaning to fix this. I'll have a crack if no-one else does.

**Comment by langevin-usgs** [comment]
Hey @cnicol-gwlogic, would be great if you could take a crack at this!  Thank you.  Feel free to ping me if you want to discuss.

**Comment by langevin-usgs** [comment]
After briefly looking at the error message, the problem appears to be in the checker.  I believe there is a simple way to turn off the checker during load (check=False).  This would be worth a shot, though I suspect there could be other issues.

**Comment by cnicol-gwlogic** [comment]
No worries @langevin-usgs - good point, check=False is the quick/easy work-around. I submitted a PR regardless, as I occasionally come across problems related to this. EVT/RCH loading still needs to be addressed for mfusg models, but this needs a bit of thought. Cheers.

**Comment by DataGeoranger** [comment]
@langevin-usgs @cnicol-gwlogic Thank you both for the tips and help. set check = False works for me when I try it on USGS usg model example. For my USG model, I had to set Forgive = True as well and then it will load model successfully. I do have a CLN package, wonder if that's why?
If the model doesn't pass a checker, will that limit me using some of the flopy utils? Because I tried to use the write_shapefile function and it didn't work either after I load the USG model. 
`mf._modelgrid.write_shapefile(os.path.join(proj_dir, '3.GIS/shp/s31_usg.shp'), epsg=epsg)`
